Pod/Classes/OTSpanContext.h

```diff
@@ -0,0 +1,32 @@
+#import <Foundation/Foundation.h>
+
+NS_ASSUME_NONNULL_BEGIN
+
+/**
+ * OTSpanContext represents Span state that must propagate to descendant Spans and across process boundaries.
+ *
+ * OTSpanContext is logically divided into two pieces: the user-level "Baggage" (see set_baggage_item and get_baggage_item) that propagates across Span boundaries and any Tracer-implementation-specific fields that are needed to identify or otherwise contextualize the associated Span instance (e.g., a <trace_id, span_id, sampled> tuple).
```

Comment:
Since Xcode doesn't support smart paragraph reformatting in comment blocks (something even `vim` does easily, FWIW), I went this route so that I didn't have to manually fix the line width over and over again.

I looked at some other libs and it seems like they enforce 80 char line widths. I will open these files in vim and make the change there. :)